app/scripts/app.js

```diff
@@ -35,7 +35,7 @@ angular.module('moodCatApp', [
         },
         controller : 'roomController'
       })
-      .state('moods.rooms', {
+      .state('rooms', {
```

Comment:
Nested state seemed to only work with the broken views. We might want to optimize this later but as we have switched away from persistent states anyway (and put everything in services) there is no need to hurry with this.